apmpackage/apm/changelog.yml

```diff
@@ -1,6 +1,8 @@
-# newer versions go on top
-#
-# change type can be one of: enhancement, bugfix, breaking-change
+- version: 8.6.0
+  changes:
+    - description: Placeholder
+      type: enhancement
+      link: https://github.com/elastic/apm-server/pull/123
```

Comment:
This should either be removed or combined with the version bump (which I created a manual PR for this time https://github.com/elastic/apm-server/pull/9156)